packages/web/src/pages/upload-page/components/TracksPreview.module.css

```diff
@@ -59,3 +59,42 @@
 .tabSlider {
   margin-bottom: 15px;
 }
+
+.containerNew {
```

Comment:
same here, but whatever you think is easiest to reason about, whether these are in a new file so its easy to remove the old code eventually.

Reply:
yea i've been updating things with the thought that we would remove old stuff later